src/reducers/migration.ts

```diff
@@ -0,0 +1,23 @@
+import { MigrationManifest, PersistedState } from 'redux-persist';
+import { List } from 'immutable';
+
+import { Debug } from '../Debug';
+import { AppState } from '.';
+import { Feed } from '../models/Feed';
+
+const migrateUnversionedToVersion0 = (state: PersistedState): AppState => {
+    Debug.log('Migrate unversioned to version 0');
+    const appState = state as AppState;
+    const version0AppState = {
+        ...appState,
+        feeds: List<Feed>(appState.feeds.toArray().map(feed => ({
```

Comment:
I've read the description in the PR, but don't really get what the issue is. Is there an actual bug - we cannot use map on Immutable.List, or is it just an error in the d.ts file? If the latter, maybe we should monkey patch it in a post install script.

Reply:
Unfortunately there are other problems with the type definitions and the immutable library itself. `map` returns an `Iterable` type that needs to be converted to a list (with `toList()`) but then the types are again wrong.

I will leave this as is for now.